<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>353</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelOperand1">
           <property name="text">
            <string>Number 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditOperand1">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="labelOperand2">
           <property name="text">
            <string>Number 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditOperand2">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="labelResult">
           <property name="text">
            <string>Result</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditResult">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButtonOne">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonTwo">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonThree">
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAdd">
           <property name="text">
            <string>+</string>
           </property>
           <property name="operation" stdset="0">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButtonFour">
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonFive">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSix">
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSub">
           <property name="text">
            <string>-</string>
           </property>
           <property name="operation" stdset="0">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="pushButtonSeven">
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonEight">
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonNine">
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonMul">
           <property name="text">
            <string>*</string>
           </property>
           <property name="operation" stdset="0">
            <number>2</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButtonZero">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonClear">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonEnter">
           <property name="text">
            <string>Enter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonDiv">
           <property name="text">
            <string>/</string>
           </property>
           <property name="operation" stdset="0">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
